index.html

```diff
@@ -12,29 +12,37 @@
 		<title>Projection Wizard</title>
 		<!--JQUERY-UI-->
 		<link rel="stylesheet" href="https://code.jquery.com/ui/1.11.0/themes/smoothness/jquery-ui.min.css" />
-		<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+		<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
 		<script src="https://code.jquery.com/ui/1.11.0/jquery-ui.min.js"></script>
 
 		<!--Leaflet AND Leaflet.PM for editing -->
-		<link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ==" crossorigin="" />
-		<script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js" integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew==" crossorigin=""></script>
+		<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+     		integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+     		crossorigin=""/>
+	 	<script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"
+     		integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo="
+     		crossorigin=""></script>
+		<!-- TODO: 
+		 Verify this is fixed as of https://github.com/geoman-io/leaflet-geoman/releases/tag/2.8.0
+		 via https://github.com/geoman-io/leaflet-geoman/pull/663
+		 via https://github.com/geoman-io/leaflet-geoman/pull/489
+		 which is all related to what we need in: https://github.com/geoman-io/leaflet-geoman/pull/464
+
+		 If fixed, we should be able to remove our local patch and go to version 2.8.0 for CSS and JS and test it out.
+		-->
 		<link rel="stylesheet" href="https://unpkg.com/leaflet.pm@2.1.0/dist/leaflet.pm.css" />
 		<!-- <script src="https://unpkg.com/leaflet.pm@2.1.0/dist/leaflet.pm.min.js"></script> -->
 		<!-- until https://github.com/codeofsumit/leaflet.pm/pull/464 is merged -->
 		<script src="lib/leaflet.pm.min.js"></script>
 
 		<!--D3-->
-		<script src="https://d3js.org/d3.v5.js"></script>
-		<script src="https://d3js.org/d3-array.v1.min.js"></script>
-		<script src="https://d3js.org/d3-fetch.v1.min.js"></script>
-		<script src="https://d3js.org/d3-geo.v1.min.js"></script>
-		<script src="https://d3js.org/d3-geo-projection.v2.min.js"></script>
-		<script src="lib/d3-geo-polygon.min.js"></script>
-		<script src="https://unpkg.com/topojson-client@3"></script>
+		<script src="https://cdn.jsdelivr.net/npm/d3@7"></script>
+		<script src="https://cdn.jsdelivr.net/npm/d3-geo@3"></script>
+		<script src="https://cdn.jsdelivr.net/npm/d3-geo-projection@4"></script>
+		<script src="https://cdn.jsdelivr.net/npm/d3-geo-polygon@1"></script>
```

Comment:
Info: `d3-geo-polygon` version 2 crashes with two-point equidistant, so I'll keep it at v1 for now.

Reply:
@jwasilgeo, sounds good. Thank you!